test/framing/FrameTransportTest.cpp

```diff
@@ -48,28 +48,6 @@ TEST(FrameTransport, CloseWithError) {
   transport->closeWithError(std::runtime_error("Uh oh"));
 }
 
-TEST(FrameTransport, SimpleEnqueue) {
```

Comment:
The whole FrameTransport class is going away so I would argue we don't have to worry about the FrameTransportTests which will go away too.